org-node/src/client.rs

```diff
@@ -125,6 +141,19 @@ pub struct Head {
     branch: String,
 }
 
+/// Project peer contains details used to track the peer for an interested project Urn.
+/// this struct is primarily used by `ProjectPeerTransmitter`
+struct ProjectPeer {
+    pub api: Peer<Signer>,
+    pub peer_info: PeerInfo<std::net::SocketAddr>,
+    pub paths: Paths,
+    pub urn: Urn,
+}
```

Comment:
Do the fields need to be public? The struct isn't.

Reply:
I would also find a better name for this that communicates its intent, eg. `TrackPeer`, since it's basically a command, right?

Reply:
6e416d0 renames, 5dfbfd6 removes `pub` fields.